lib/support_rota_client.rb

```diff
@@ -0,0 +1,22 @@
+class SupportRotaClient
+  def initialize
+    @api_base = if Rails.env.development? || Rails.env.test?
+                  "http://localhost:3000"
+                else
+                  "https://dxw-support-rota.herokuapp.com"
+                end
```

Comment:
Does this mean that you need to be running the support rota app locally in order to run this project now?

Reply:
Fixed the link now, although I think I'll be "descoping" moving it to an env var for the "Next" part of the backlog. 😬